frontend/src/pages/devices/components/LockCode.tsx

```diff
@@ -157,7 +168,7 @@ export const LockCode = (props: Props) => {
             type="datetime-local"
             defaultValue={format(startAt, "yyyy-MM-dd'T'HH:mm")}
             onChange={(evt) => setStartAt(parseISO(evt.target.value))}
-            disabled={!!props.managedLockCode}
+            disabled={enabledLogic()}
```

Comment:
The name of the function doesn't seem intuitive... we're `disabled` if `enabledLogic`.